src/tools/rustfmt/tests/target/issue-4791/issue_4928.rs

```diff
@@ -36,7 +36,7 @@ pub struct BufferAttr {
     /// odio tempor.
     ///
     /// Lacinia at quis risus sed vulputate odio ut enim. Etiam erat velit scelerisque in dictum.
-    /// Nibh tellus molestie nunc non blandit massa enim nec. Nascetur ridiculus mus mauris vitae.
+    /// Nibh tellus molestie nunc non blandit massa enim nec. Nascetur ridiculous mus mauris vitae.
```

Comment:
The content of this doc comment is [Lorem ipsum](https://en.wikipedia.org/wiki/Lorem_ipsum) which is Latin so I believe the original version is correct. Please back out this change.

Reply:
I'll also add that in general the rustfmt team prefers these types of changes to be made directly to https://github.com/rust-lang/rustfmt, and for that reason I'd also like to revert this change.